cromwell_tools/cromwell_api.py

```diff
@@ -128,29 +130,33 @@ def health(cls, auth, raise_for_status=False):
 
     @classmethod
     @retry(reraise=True, wait=wait_exponential(multiplier=1, max=10), stop=stop_after_delay(20))
-    def submit(cls, auth, wdl_file, inputs_file, options_file=None, inputs_file2=None, zip_file=None,
-               collection_name=None, label=None, validate_labels=False, on_hold=False):
-        # TODO: Allow more inputs files rather 2 to be consistent with the Cromwell API.
+    def submit(cls, auth, wdl_file, inputs_file, options_file=None, dependencies=None,
+               label_file=None, collection_name=None, on_hold=False, validate_labels=False, **inputs_file_n):
```

Comment:
This could also take `inputs_files` as a list, defaulted to `None`. The submit CLI could then take an arbitrary number of `--inputs-file` options, with `dest=inputs_files` and `action='append'`.

Reply:
👍